README.md

```diff
@@ -41,6 +41,23 @@ You can now execute the command line program `org-formation`. try:
 > org-formation --help
 ```
 
+### Docker
+
+If you choose, you can run org-formation in a docker container:
+
+```sh
+# Set the AWS_PROFILE environment variable and pass it to the container
+> AWS_PROFILE=example
+# Run the container
+> docker run --rm -it -v $HOME/.aws:/root/.aws:ro -v $PWD:/workdir -w /workdir -e AWS_PROFILE <TODO:CONFIRM_DOCKER_HUB_USER>/org-formation-cli
```

Comment:
we can the right repository here: `orgformation/org-formation-cli`

Reply:
Updated with official name from hub.docker.com.